packages/contracts/contracts/L2/predeploys/iOVM_L1BlockNumber.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+/**
+ * @title iOVM_L1BlockNumber
+ */
+interface iOVM_L1BlockNumber {
```

Comment:
@smartcontracts Do we want to standardize on not having interfaces? You mentioned that, I do know that its convenient to import them only when the solidity pragma is loose, but it is very easy for devs to generate their own interfaces with `cast interface <address verified on etherscan or path to artifact>`